operations/iuf/IUF.md

```diff
@@ -854,6 +854,14 @@ The following actions may be useful if errors are encountered when executing `iu
    check whether CFS is unable to start the configuration again for the node due to any other issue. Rectify the problem by referring to the
    [CFS troubleshooting guide](../../operations/configuration_management/Troubleshoot_CFS_Sessions_Failing_to_Start.md)
 
+ 1. IUF workflow does not start after the re-installation of Keycloak.
+
+    - IUF waits for workflow to start. This step might not be completing because cray-nls stores the client-id and client-secret which is initialized when cray-nls pod is started if client_secret is modified after initializing such an error is possible.
+
+    - Look at the cray-nls pod logs. It should show that it is not able to validate "admin-client" id with keycloak as It was returning "unauthorized_client" new token request from cray-nls backend for "admin-client" id
```

Comment:
Can you provide the exact error seen in the logs here? I think it would be helpful to provide exact output.
Also, you have a capitalized "I" that should be lowercase in: "keycloak as It was".